### PR TITLE
Rhmap 17468 - lang review and unmerged changes from master

### PR DESCRIPTION
**Pull request by pmacko1**
https://issues.jboss.org/browse/RHMAP-17468

### CODE REVIEW

File: admin/create-osx-user.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ GROUP_ID=20
 ############### end of parameters
 ###############
 
-echo "IMPORTANT: this script will now create user ${USERNAME} with password ${PASSWORD}."
+echo "IMPORTANT: this script will now create user ${USERNAME} with password ${PASSWORD} and add him to sudoers list."
```
Comment (laurafitzgerald) [inline admin/create-osx-user.sh]: him -> them

File: admin/create-osx-user.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ GROUP_ID=20
 ############### end of parameters
```
Comment (laurafitzgerald) [inline admin/create-osx-user.sh]: Any changes to this file would need to be tested/approved as we are linking to this from the documentation for the customer to use.

**Comment by laurafitzgerald** [comment]
@pmacko1 I wasn't clear on this pr but I see now it is a cherry-pick of changes to the FH-v4.5 branch. Is that correct?

**Comment by pmacko1** [comment]
@laurafitzgerald i'm not merging changes which are not mine. Closing.
see https://issues.jboss.org/browse/RHMAP-17468 pls

**Pull request closed without merging**